stats/models.py

```diff
@@ -280,31 +281,29 @@ def agg_timeseries(
 
 
 class DailyStat(models.Model):
-    date = models.DateField(help_text="Le jour de la statistique")
+    date = models.DateField(help_text=_("Date of statistics"))
     # answers
-    question_answer_count = models.PositiveIntegerField(default=0, help_text="Le nombre de questions répondues")
+    question_answer_count = models.PositiveIntegerField(default=0, help_text=_("Answered questions count"))
     question_public_answer_count = models.PositiveIntegerField(
-        default=0, help_text="Le nombre de questions publiques répondues"
+        default=0, help_text=_("Answered public questions count")
     )
     question_answer_from_quiz_count = models.PositiveIntegerField(
-        default=0, help_text="Le nombre de questions répondues au sein de quizs"
-    )
-    quiz_answer_count = models.PositiveIntegerField(default=0, help_text="Le nombre de quizs répondus")
-    quiz_public_answer_count = models.PositiveIntegerField(
-        default=0, help_text="Le nombre de quizs publiques répondus"
+        default=0, help_text=_("Answered questions within quiz count")
     )
+    quiz_answer_count = models.PositiveIntegerField(default=0, help_text=_("Answered quizzes count"))
```

Comment:
```suggestion
    quiz_answer_count = models.PositiveIntegerField(default=0, help_text=_("Answered quizs count"))
```
Actuellement j'ai opté pour utiliser "quizs"